TDLib/Bindings/Receiver.cs

```diff
@@ -46,19 +46,30 @@ private void ProcessEvents()
             var ct = _cts.Token;
             while (!ct.IsCancellationRequested)
             {
-                var data = _tdJsonClient.Receive(0.1);
-
-                if (!string.IsNullOrEmpty(data))
+                string data = null;
+                try
                 {
-                    var structure = JsonConvert.DeserializeObject<TdApi.Object>(data, _converter);
-                    
-                    Received?.Invoke(this, structure);
+                    data = _tdJsonClient.Receive(0.1);
 
-                    if (structure is TdApi.Update.UpdateAuthorizationState update)
+                    if (!string.IsNullOrEmpty(data))
                     {
-                        AuthorizationStateChanged?.Invoke(this, update.AuthorizationState);
+                        var structure = JsonConvert.DeserializeObject<TdApi.Object>(data, _converter);
+
+                        Received?.Invoke(this, structure);
+
+                        if (structure is TdApi.Update.UpdateAuthorizationState update)
+                        {
+                            AuthorizationStateChanged?.Invoke(this, update.AuthorizationState);
+                        }
                     }
                 }
+                catch (Exception ex)
```

Comment:
I don't think printing to Console from the library is a good solution. Maybe event would fit better in this case

Reply:
Added.